python/GafferImageTest/ImageWriterTest.py

```diff
@@ -745,13 +746,23 @@ def __testMetadataDoesNotAffectPixels( self, ext, overrideMetadata = {}, metadat
 			if metaName in metadataToIgnore :
 				continue
 			self.assertTrue( metaName in misledReaderMetadata.keys(), "Writer Metadata missing expected key \"{}\" set to \"{}\" : {}".format(metaName, str(expectedMetadata[metaName]), ext) )
-			self.assertEqual( expectedMetadata[metaName], misledReaderMetadata[metaName], "Metadata does not match for key \"{}\" : {}".format(metaName, ext) )
+
+			if metaName == "DateTime" :
+				dateTimeDiff = datetime.datetime.strptime( str( expectedMetadata[metaName] ), "%Y:%m:%d %H:%M:%S" ) - datetime.datetime.strptime( str( misledReaderMetadata[metaName] ), "%Y:%m:%d %H:%M:%S" )
+				self.assertLessEqual( abs( ( dateTimeDiff.days * 86400 ) + dateTimeDiff.seconds ), 1 )
```

Comment:
Can we use `self.assertLessEqual( abs( dateTimeDiff ), datetime.timedelta( seconds=1 )` here instead?